sw/host/provisioning/ft_lib/BUILD

```diff
@@ -2,13 +2,14 @@
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
 
-load("@rules_rust//rust:defs.bzl", "rust_library")
+load("@rules_rust//rust:defs.bzl", "rust_library", "rust_test")
 
 package(default_visibility = ["//visibility:public"])
 
 rust_library(
     name = "ft_lib",
     srcs = ["src/lib.rs"],
+    data = ["//sw/device/silicon_creator/manuf/keys/fake:fake_ca.pem"],
```

Comment:
Ah, you want to add this as a data dep to the binary rule, not the library rule, by adding it either:
- https://cs.opensource.google/opentitan/opentitan/+/master:sw/host/provisioning/ft/BUILD;drc=30d7e787c753caaa03fe68a4a70da1bbcbc1d96f;l=21 or
- https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/silicon_creator/manuf/provisioning_inputs.bzl;drc=1d40a002b5d5ceb333cb26361a0c7a15db6e92f4;l=22